setup/db/db/schema-4920to41000.sql

```diff
@@ -234,3 +234,26 @@ WHERE (o.cpu is null AND o.speed IS NULL AND o.ram_size IS NULL) AND
 
 -- CLOUDSTACK-9827: Storage tags stored in multiple places
 DROP VIEW IF EXISTS `cloud`.`storage_tag_view`;
+
+DROP TABLE IF EXISTS `cloud`.`netscaler_servicepackages`;
+CREATE TABLE `cloud`.`netscaler_servicepackages` (
```

Comment:
Actually, we followed the existing approach. The scenario you told is also valid but I wonder if anyone would really be using a PR branch in production without community approval.